src/datanode/src/mock.rs

```diff
@@ -91,7 +91,7 @@ impl Instance {
     pub async fn with_mock_meta_server(opts: &DatanodeOptions, meta_srv: MockInfo) -> Result<Self> {
         let object_store = new_object_store(&opts.storage).await?;
         let log_store = create_local_file_log_store(opts).await?;
-        let meta_client = Arc::new(mock_meta_client(meta_srv, opts.node_id).await);
+        let meta_client = Arc::new(mock_meta_client(meta_srv, 42).await);
```

Comment:
Why hard-code？

Reply:
I found this will break some test if node id is harcoded.